### PR TITLE
feat: Apply wasmd v0.29.1

### PR DESCRIPTION
**Pull request by zemyblue**
## Description

refers: #4 

* bump up wasmd from v0.27.0 to v0.29.1. [full changelogs](https://github.com/CosmWasm/wasmd/blob/v0.29.1/CHANGELOG.md)
* replace `line/lbm-sdk/x/ibc` to `line/ibc-go v3.3.1`

## Motivation and context

Bump up wasmd v0.29.1 to apply wasmvm v1.1.1 and cosmwasm v1.1.9.

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by zemyblue** [comment]
I am waiting for ibc-go v3.3.0 to apply to `line/ibc-go`.

**Comment by tkxkd0159** [comment]
`.github/workflows/proto-buf-publisher.yml` seems empty. Does it need to add in this PR?

**Comment by zemyblue** [comment]
> `.github/workflows/proto-buf-publisher.yml` seems empty. Does it need to add in this PR?

We don't need it. But It looks like it was added during the merge process. I'll remove it.

**Review by tkxkd0159 — APPROVED** [review]
LGTM

File: docker/setup_and_run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+./setup_wasmd.sh "$@"
+./run_wasmd.sh
```
Comment (shiki-tak) [inline docker/setup_and_run.sh:4]: nit: No newline at end of file

File: proto/buf.gen.doc.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+version: v1
+plugins:
+  - name: doc
+    out: ../docs/proto
+    opt: ../docs/proto/protodoc-markdown.tmpl,proto-docs.md
```
Comment (shiki-tak) [inline proto/buf.gen.doc.yml:5]: nit: No newline at end of file

File: proto/buf.gen.gogo.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+version: v1
+plugins:
+  - name: gocosmos
+    out: ..
+    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types
+  - name: grpc-gateway
+    out: ..
+    opt: logtostderr=true,allow_colon_final_segments=true
```
Comment (shiki-tak) [inline proto/buf.gen.gogo.yml:8]: nit: No newline at end of file

**Comment by zemyblue** [comment]
CI error is due to a decrease in coverage percent of Covercov. This is because the coverage of the code that was versioned up in origin was reduced. Please ignore it and review it.

**Review by shiki-tak — APPROVED** [review]
LGTM

**Pull request merged**